be/src/storage/tablet_updates.cpp

```diff
@@ -1986,6 +1987,7 @@ void TabletUpdates::get_tablet_info_extra(TTabletInfo* info) {
         if (_edit_version_infos.empty()) {
             LOG(WARNING) << "tablet delete when get_tablet_info_extra tablet:" << _tablet.tablet_id();
         } else {
+            min_version = _edit_version_infos[0]->version.major();
```

Comment:
How about the non-primary key table?

Reply:
not support yet

Reply:
It's better to make the behavior of all models consistent. Otherwise, it will be a trick to remember the difference.

Reply:
Does you can use the first singleton delta version to be minReadableVersion of non-primary table?

Reply:
It may be too restrictive, for example:
```
[1-100], [100, 200], [201], [202], [203]
```
If `minReadableVersion` == 201,  but version `100` `200` still can be read

Reply:
Added support for non-primary key tablets